### PR TITLE
Nft scaffold initial docs

### PR DESCRIPTION
**Pull request by hu3man**
This pr adds a bare-bones doc for the nft-scaffold and updates the other scaffold docs to take into account the implicit setup script being run

### CODE REVIEW

File: website/docs/scaffolds/nft-scaffold.mdx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+:::warning
+The NFT-Scaffold project is currently in alpha. It is not ready for production use and will be undergoing frequent updates as we work to implement Taqueria State and other features that have not yet been released
+:::
+
+## Project Setup 
+
+1. Scaffold the project by running:
+
+```bash
+taq scaffold https://github.com/ecadlabs/taqueria-scaffold-quickstart quickstart-project
```
Comment (mweichert) [inline website/docs/scaffolds/nft-scaffold.mdx:22]: The should be `https://github.com/ecadlabs/taqueria-scaffold-nft`.

The output directory should be `nft-quickstart`.

File: website/docs/scaffolds/nft-scaffold.mdx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+The NFT-Scaffold project is currently in alpha. It is not ready for production use and will be undergoing frequent updates as we work to implement Taqueria State and other features that have not yet been released
+:::
+
+## Project Setup 
+
+1. Scaffold the project by running:
+
+```bash
+taq scaffold https://github.com/ecadlabs/taqueria-scaffold-quickstart quickstart-project 
+```
```
Comment (mweichert) [inline website/docs/scaffolds/nft-scaffold.mdx:23]: Perhaps we should add a step here to change to the `nft-quickstart` directory?

File: website/docs/scaffolds/nft-scaffold.mdx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+pinataJwtToken=eyJhbGc...
+```
+
+5. Start the Taqueria project by running the start script:
+
+```shell
+npm run start:taqueria:local
+```
+
+6. Apply the Provisions
```
Comment (mweichert) [inline website/docs/scaffolds/nft-scaffold.mdx:41]: Perhaps we should add:

> NOTE: Taqueria's provisioning system is work-in-progress. However, the provisioning system prototype will closely resemble the version that will be shipped with Taqueria later this year.

File: website/docs/scaffolds/taco-shop.mdx [hunk 4]
```diff
@@ -19,7 +19,7 @@ In a rush? You can follow the steps below to get up and running immediatley
 
 1. `taq scaffold https://github.com/ecadlabs/taqueria-scaffold-taco-shop taco-shop`
 2. `cd taco-shop`
-3. `npm run setup`
+3. `npm run start`
 
 ## Overview
```
Comment (mweichert) [inline website/docs/scaffolds/taco-shop.mdx:24]: Although not necessary for merging this in, in the future we should probably add a heading and section for provisioning.

**Review by mweichert — APPROVED** [review]

File: website/docs/scaffolds/nft-scaffold.mdx [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+## Project Setup 
+
+1. Scaffold the project by running:
+
+```bash
+taq scaffold https://github.com/ecadlabs/taqueria-scaffold-quickstart quickstart-project 
+```
+
+2. Create the file `./taqueria/.env
```
Comment (alexzbusko) [inline website/docs/scaffolds/nft-scaffold.mdx:25]: This is to further Michael's comment on making sure they are in the scaffolded project. Can we be explicit that this is the taqueria directory within the NFT scaffold?

File: website/docs/scaffolds/nft-scaffold.mdx [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+- `taqueria`
+
+    - Everything related to the contract
+    - `taqueria/.taq`
+        - Taqueria config folder, including setup for all required plugins
+    - `taqueria/contracts`
+        - The contract .ligo code
+    - `taqueria/artifacts`
+        - The compiled contract (*.tz file)
+    - `taqueria/typings`
```
Comment (alexzbusko) [inline website/docs/scaffolds/nft-scaffold.mdx:72]: I think this is generally referred to as `types`. In the actual folder itself it is types

File: website/docs/scaffolds/taco-shop.mdx [hunk 7]
```diff
@@ -114,26 +121,25 @@ The intended workflow for this project is as follows:
...
+taq originate -e ithacanet
 ```
 
 This should return the address of the contract on the testnet which looks like this:
 ```shell
 ┌────────────────┬──────────────────────────────────────┬────────────────┐
 │ Contract       │ Address                              │ Destination    │
 ├────────────────┼──────────────────────────────────────┼────────────────┤
-│ hello-tacos.tz │ KT1EHAXNQ1oeh2ZNTkvvcU6PfzQvGPLsgJx3 │ ghostnet       │
+│ hello-tacos.tz │ KT1EHAXNQ1oeh2ZNTkvvcU6PfzQvGPLsgJx3 │ ithaca_network │
```
Comment (russellmorton) [inline website/docs/scaffolds/taco-shop.mdx:142]: Should this be ghostnet?

Comment (hu3man) [reply website/docs/scaffolds/taco-shop.mdx:142]: Yep, thanks!

**Review by russellmorton — APPROVED** [review]

**Pull request merged**